_plugins/related_posts.rb

```diff
@@ -1,65 +0,0 @@
-module Jekyll
```

Comment:
@gboone this doesn't appear to be used anywhere. I removed it and the related posts section still works.

[Check it out](https://federalist.18f.gov/preview/18F/18f.gsa.gov/same-headers/2016/07/29/live-streamed-demos-for-exponential-transparency-and-information-sharing/)